src/main/java/teammates/common/datatransfer/FeedbackMsqQuestionDetails.java

```diff
@@ -417,8 +419,10 @@ public String getQuestionResultStatisticsHtml(List<FeedbackResponseAttributes> r
         
         int numChoicesSelected = 0;
         for (FeedbackResponseAttributes response : responses) {
-            List<String> answerStrings = ((FeedbackMsqResponseDetails) response.getResponseDetails()).getAnswerStrings();
-            Boolean isOtherOptionAnswer = ((FeedbackMsqResponseDetails) (response.getResponseDetails())).isOtherOptionAnswer();
+            List<String> answerStrings =
+                    ((FeedbackMsqResponseDetails) response.getResponseDetails()).getAnswerStrings();
+            boolean isOtherOptionAnswer =
+                    ((FeedbackMsqResponseDetails) (response.getResponseDetails())).isOtherOptionAnswer();
```

Comment:
Not directly related to your changes, but it looks like there are some unnecessary parentheses here =P